src/client/SignIn.ts

```diff
@@ -14,9 +14,10 @@ import { Big } from "big.js";
  *
  * @param {string} username - This field is required.
  * @param {string} password - This field is required.
+ * @param {string} useAuthToken - This field is optional.
```

Comment:
Shouldn't this be `@param {boolean} useAuthToken - This field is optional`

Reply:
Fixed 👍